src/main/java/bio/ferlab/fhir/schema/definition/SchemaDefinition.java

```diff
@@ -30,10 +30,10 @@ public SchemaDefinition(String schemaName) {
     public SchemaDefinition(String schemaName, String profileName, String[] extensionNames) {
         this(schemaName);
         if (StringUtils.isNotBlank(profileName)) {
-            setProfile(FhavroConverter.loadProfile(profileName));
+            setProfile(Fhavro.loadProfile(profileName));
             if (extensionNames != null && extensionNames.length > 0) {
```

Comment:
(Optional): if CollectionUtils is available you could use the same pattern as with Strings at line 32 => CollectionUtils.isNotEmpty